.github/scripts/instr-count-benchmark/upload.py

```diff
@@ -0,0 +1,121 @@
+"""Scribe Uploader for Pytorch Microbenchmark Data"""
+import argparse
+import json
+import hashlib
+import sys
+import time
+
+import numpy as np
+import torch
+
+from scripts.upload_scribe import ScribeUploader
+
+
+VERSIONS = [
+    'b984f1ebad546ed3ee612244b8c72e53',
+]
+
+
+class PytorchMicrobenchmarkUploader(ScribeUploader):
+    def __init__(self):
+        super().__init__('perfpipe_pytorch_microbenchmarks')
+        self.schema = {
+            'int': [
+                'time',
+                'start_time',
+                'end_time',
+                'benchmark_version',
+                'task_number',
+
+                # We don't need to record as many summary statistics for counts
+                # as we do for times, because we retain them all in
+                # `counts_json`.
+                'count_min', 'count_max',
+                'count_p25', 'count_median', 'count_p75',
+            ],
+            'normal': [
+                # Unique ID for the run.
+                'run_id',
+
+                # Metadata for ad-hoc analysis.
+                'task_key', 'counts_json',
+
+                # TODO: More environment info.
+                'pytorch_version',
```

Comment:
We should also record the cuda version here.

Reply:
Done.